modules/hub/ops/prod.dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM connext_builder:dev
 
-COPY node_modules node_modules
-COPY ops ops
-COPY dist dist
+COPY modules/client /client
```

Comment:
This is temporary, once we have a better way to keep the connext module on npm up-to-date, we won't need to manually copy the client module into the hub's prod-mode image